### ISSUE TITLE
double converted to int

### ISSUE THREAD

**Issue by Hollz** [javiercbk/json_to_dart#28]
{"price":830.1}  -> double price;

{"price":830.0}  -> int price  ？

**Comment by javiercbk** [comment]
This is a dart issue documented in the project caveats section:

`Dart automatically casts doubles without decimals (like 10.0) to int. Use doubles with at least one decimal number.`

**Comment by Hollz** [comment]
Thank you

**Comment by Torlinone** [comment]
@javiercbk #36 , that fixed ,but you json formatter still convert json like:
```json
{
    "numberO": 1.0,
    "numberT": 1
}
```
to
```json
{
    "numberO": 1,
    "numberT": 1
}
```

please check again. thank you.

**Comment by javiercbk** [comment]
I think this is an issue with the JSON formatter.

I format the json in order to easily pin point warnings for the user, but it seems that the formatter is removing that trailing zero

**Comment by Torlinone** [comment]
@javiercbk yes, it's JSON formatter's issus. maybe this JSON formatter is unnecessary， you can remove it...
